.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": [
+    "twilio-react"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2019,
+    "sourceType": "module", 
+    "ecmaFeatures": { 
+    "jsx": true
+   }
+  },
+  "rules": {
+    "no-console": "off",
```

Comment:
I'm curious what the source was for these disabled rules? Did you copy them from another configuration in a different repo? 

I don't have a major issue with any of the specific rules being disabled, just that I think some of them are enabled in the Flex plugin so wondering what the thinking was behind disabling these particular ones here?